invenio_accounts/ext.py

```diff
@@ -198,6 +198,9 @@ def init_app(self, app, sessionstore=None, register_blueprint=True):
                 app.extensions['security'].confirm_register_form, app
             )
 
+        app.extensions['security'].login_form = login_form_factory(
```

Comment:
Maybe it could be better to pass the [login_form as parameter](https://github.com/mattupstate/flask-security/blob/develop/flask_security/core.py#L474) when you [initialize the security app](https://github.com/inveniosoftware/invenio-accounts/blob/master/invenio_accounts/ext.py#L189)?